docs/source/files/bootstrap-hypervisor.yml

```diff
@@ -4,6 +4,7 @@
     _user: "{{ remote_user | default('zuul') }}"
   tasks:
     - name: Create user on remote host
```

Comment:
sounds weird, since:
- the doc mentions "password root access"
- that playbook is the one creating the rootless user + sudo

so basically.... you mostly don't use that playbook... ? I had the same reflection with a DS equivalent patch...

Reply:
The usecases are:
- you have a baremetal "cloud" 
- you run nested virt in some cloud
 ... and the server you requested is provisioned - you are given access via the "cloud-user" of the image deployed. You have SSH access, you have sudo - but you cannot login as root.

It is possible to just use the `cloud-user`. But with that follows the editing of stuff in notes/chat with co-workers that uses the default `zuul` user.

Reply:
oh. ok. heh, makes sense.
Note that my own baremetal doesn't have "zuul" user - that's how I catch (many) issues with paths, facts and so on.